.github/workflows/nightly.yml

```diff
@@ -0,0 +1,56 @@
+name: Kyuubi Nightly
+
+on:
+  schedule:
+    - cron: 0 23 * * *
```

Comment:
What is the time for spark nightly build?

Reply:
midnight, and should we change it to 1:00 am?

Reply:
Have look on Spark GA list, nightly build usually cost 1~2 hours, and sometime over 3 hours, maybe we should start at 4:00 am.